cli/tests/upgrade.bats

```diff
@@ -186,6 +186,26 @@ teardown() {
   assert_equal "$curr_lock_hash" "$prev_lock_hash"
 }
 
+@test "upgrade for flake installable" {
+  "$FLOX_BIN" init
+  cat <<EOF | "$FLOX_BIN" edit -f -
+  version = 1
+
+  [install]
+  hello.flake = "github:NixOS/nixpkgs#hello"
```

Comment:
I think it would be good to consolidate all our tests that are slow and depend on GitHub in one place. Not sure yet what would be best for that

Reply:
Not sure what you mean by "depend on GitHub in one place"

Reply:
Like have all of our tests that fetch a flake from github, evaluate it, and build store paths from it all in a single file. Right now I think we fetch one revision in buildenv tests, another in install, and a 3rd here

Reply:
And if all of that fetching is too slow, we may need to change some of these tests to depend on mock data instead of GitHub